instancetypes/gn/1/gn1.yaml

```diff
@@ -24,4 +24,4 @@ metadata:
 spec:
   gpus:
     - name: "gpu1"
-      deviceName: "nvidia.com/A400"
+      deviceName: "nvidia.com/gpu"
```

Comment:
I'd not switch to a generic one, instead ideally this is useable out of the box.

Can we get NVIDIA to tell us what the most relevant GPU for this scenario is and use the resource here?
Or the top 3.

Reply:
@rthallisey do you happen to know who can help here?

Reply:
> I'd not switch to a generic one, instead ideally this is useable out of the box.

The current choice of A400 or any other specific device name makes this unusable out of the box for the vast majority of users even with the device plugin configured and GPUs in the cluster IMHO.

> Can we get NVIDIA to tell us what the most relevant GPU for this scenario is and use the resource here?
> Or the top 3.

I think we should provide an example kustomise layer on top of common-instancetypes that sets a specific Nvidia device but default to the generic default vgpu device, assuming this works with KubeVirt.